Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM icr.io/appcafe/open-liberty:kernel-slim-java11-openj9-ubi
+ARG VERSION=1.0
+ARG REVISION=SNAPSHOT
+
+LABEL \
+  org.opencontainers.image.authors="Your Name" \
+  org.opencontainers.image.vendor="IBM" \
+  org.opencontainers.image.url="local" \
+  org.opencontainers.image.source="https://github.com/OpenLiberty/guide-getting-started" \
+  org.opencontainers.image.version="$VERSION" \
+  org.opencontainers.image.revision="$REVISION" \
+  vendor="Open Liberty" \
+  name="system" \
+  version="$VERSION-$REVISION" \
+  summary="The system microservice from the Getting Started guide" \
```

Comment:
```suggestion
  summary="Sample microservice running on Open Liberty runtime that leverages MicroProfile." \
```

Reply:
I didn't use micro service, but just called it an app because I don't think it is really a microservice.